csharp/test/Ice/dictMapping/ServerAMD.cs

```diff
@@ -4,21 +4,21 @@
 
 using Test;
 
-namespace ZeroC.Ice.Test.DictMapping.AMD
+namespace ZeroC.Ice.Test.DictMapping
 {
-    public class Server : TestHelper
+    public class ServerAMD : TestHelper
```

Comment:
Would `ServerAsync` be more consistent with the servant name?

Reply:
Even though it would be more consistent with the servant name, it would be inconsistent with how we name our test files in other similar cases. Like for AMI, we always use it as a suffix: `TwowaysAMI`. Of course, we could change all of them.

But changing our names to use 'Async' instead of 'AMD' wouldn't stop at just the naming of the servers, we use the AMD suffix all over the place in tests.

Reply:
Ok, let's keep AMD.